spec/swagger_spec.rb

```diff
@@ -29,7 +29,8 @@
 
       it "matches the routes" do
         redirect_routes = [{:path => "#{path_prefix}/#{app_name}/v0/*path", :verb => "DELETE|GET|OPTIONS|PATCH|POST"}]
-        expect(rails_routes).to match_array(swagger_routes + redirect_routes)
+        internal_api_routes = [{:path => "/internal/v0/*path", :verb => "GET"}, {:path=>"/internal/v0.0/authentications/:id", :verb=>"GET"}]
+        expect(rails_routes).to match_array(swagger_routes + redirect_routes + internal_api_routes)
```

Comment:
Does this mean that we are not creating a swagger spec with the internal routes? Is the intention for us not to be able to generate a client for those? Or maybe we should create a separate spec?